scripts/sync.py

```diff
@@ -147,7 +154,12 @@ def do(args):
 
     # transfer new files after all sync attempts are done
     if args.lochness_sync_send:
-        lochness_to_lochness_transfer(Lochness)
+        if args.s3:
+            lochness_to_lochness_transfer_s3(Lochness)
+        elif args.rsync:
```

Comment:
Can `s3` and `rsync` both be `True` when `lochness_sync_send` is `True`?

Reply:
yes, if both `True`, it will run `s3` transfer.

Reply:
Sorry, the question was can this happen `lochness_sync_send=True and s3=True and rsync=True`? If yes, we understand that we are prioritizing `s3=True`.